save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt

```diff
@@ -18,13 +19,22 @@ import react.dom.*
 import react.table.columns
 
 import kotlinx.html.ButtonType
+import kotlinx.html.hidden
+
+/**
+ * `Props` retrieved from router
+ */
+@Suppress("MISSING_KDOC_CLASS_ELEMENTS")
+external interface CreationViewProps : PropsWithChildren {
```

Comment:
```suggestion
external interface CreationViewProps : Props {
```
I think we had `PropsWithChildren` here by mistake, it's not necessary